src/server.ts

```diff
@@ -119,77 +144,86 @@ app.post("/killConnection", async (req, res) => {
     webParticipant &&
     req.body.participantId == webParticipant.id
   ) {
-    // TODO - move the null check inside of the functions
     console.log("deallocating all configured resources on exit");
     await killSipUriLeg(bridgeParticipant);
     await deleteParticipant(bridgeParticipant);
     await deleteParticipant(webParticipant);
     await deleteSession();
+    clientWs.close();
```

Comment:
You might want to put an existence check around this:

if (clientWs) {
   clientWs.close()
}

Reply:
thanks